mobile/src/components/Menu.js

```diff
@@ -152,6 +176,16 @@ class Menu extends Component {
     return (
       <View style={{ flex: 1 }}>
         <Text style={styles.title}>Menu</Text>
+        {!this.props.isConnected && this.props.lastOnlineTime !== null
+        && this.props.lastOnlineTime !== undefined
+          &&
+          <Text style={[styles.lastOnlineText]}>
+            <Icon name="info-circle" />
+            {' Last online at '}
+            {`${new Date(this.props.lastOnlineTime).toLocaleTimeString()} ${new Date(this.props.lastOnlineTime).toLocaleDateString()}`}
```

Comment:
Can't you just do:
{\`Last online at  ${new Date(this.props.lastOnlineTime).toLocaleTimeString()} ${new Date(this.props.lastOnlineTime).toLocaleDateString()}`}

Reply:
Yup, done